draft-ietf-rats-corim.md

```diff
@@ -1466,6 +1466,10 @@ CoRIM-based data structures define an external representation of Conceptual Mess
 Appraisal processing describes both mapping transformations and Verifier reconciliation ({{sec-verifier-rec}}).
 Non-CoRIM-based data structures require mapping transformation, but these are out of scope for this document.
 
+If a Verifier requires CoBOMs, then the CoRIM inputs indicated explicitly by the CoBOM are the only appropriate and required CoRIMs.
+If the Verifier does not require CoBOMs, then the CoRIM inputs consist of the knowledge base of CoRIMs available to the Verifier at the beginning of appraisal.
+The knowledge base state SHOULD have an immutable label for attestation results to have auditable provenance.
```

Comment:
I like framing this in terms of "CoRIM knowledge base".

However, I'd scale this one up to also cover the "Verifier requires CoBOMs" case, e.g.:

> The CoRIMs available to the Verifier at the beginning of appraisal constitute the Verifier’s "knowledge base state."
> If a Verifier requires CoBOMs, its knowledge base state is made of the CoRIM inputs indicated explicitly by the CoBOMs.
> If the Verifier does not require CoBOMs, the knowledge base state consists of all the CoRIMs available to the Verifier.
> The knowledge base state SHOULD have an immutable label for attestation results to have auditable provenance.